examples/example.go

```diff
@@ -81,8 +81,7 @@ func main() {
 	for i := 0; i < 10; i++ {
 		args = kinesis.NewArgs()
 		args.Add("StreamName", streamName)
-		args.AddData([]byte(fmt.Sprintf("Hello AWS Kinesis %d", i)))
-		args.Add("PartitionKey", fmt.Sprintf("partitionKey-%d", i))
+		args.AddRecord([]byte(fmt.Sprintf("Hello AWS Kinesis %d", i)), fmt.Sprintf("partitionKey-%d", i))
```

Comment:
How do you feel about adding temp vars so the parameters are named:

```go
msg := []byte(fmt.Sprintf("Hello AWS Kinesis %d", i))
partitionKey := fmt.Sprintf("partitionKey-%d", i)

args.AddRecord(msg, partitionKey)
```

Or maybe just a line-break to make a little more readable:

```go
args.AddRecord(
  []byte(fmt.Sprintf("Hello AWS Kinesis %d", i)), 
  fmt.Sprintf("partitionKey-%d", i),
)
```